### PR TITLE
Tuzla adapter

### LINKED ISSUE #276: Tuzla, Bosnia AQ data
We received a request while in Tuzla, Bosnia to add this data source:

http://monitoringzrakatk.info/bkc-today.html

If someone is interested in writing an adapter for this source and would like to also connect to the individual who recommended the source, let me know and I will connect you by email.

### PR DESCRIPTION
**Pull request by dolugen**
closes #276

### CODE REVIEW

**Comment by jflasher** [comment]
@dolugen Thanks! If you're all done, I can go ahead and review soon?

**Comment by dolugen** [comment]
@jflasher Yes, please!

File: adapters/tuzlanski.js [hunk 1]
```diff
@@ -0,0 +1,131 @@
+'use strict';
+
+import { REQUEST_TIMEOUT } from '../lib/constants';
+import { default as baseRequest } from 'request';
+const request = baseRequest.defaults({timeout: REQUEST_TIMEOUT});
+import cheerio from 'cheerio';
+import Coordinates from 'coordinate-parser';
```
Comment (jflasher) [inline adapters/tuzlanski.js]: What do you think about using http://turfjs.org/docs/#point for this? We're already using Turf in fetch (https://github.com/openaq/openaq-api/blob/develop/package.json#L41) and would be good to not add another library into the mix.

Comment (dolugen) [inline adapters/tuzlanski.js]: I'm only using `coordinate-parser` to convert this format: `44°31´55´´N` to decimal degrees. I don't see how Turf does that.

**Comment by dolugen** [comment]
> So much hackiness

I blame the markup.

File: adapters/tuzlanski.js (hunk 1 above)
Comment (jflasher) [inline adapters/tuzlanski.js]: Nope, you're right, wasn't really thinking over what the exact use case was.

**Pull request merged**